test/unit/test_graph.py

```diff
@@ -137,6 +138,7 @@ def _mock_hook_path(self):
                 searched_path='.',
                 project_root=os.path.normcase(os.getcwd()),
                 relative_path='dbt_project.yml',
+                modification_time=time.time(),
```

Comment:
nit: I'd try to not use `time.time()` during testing, this value will change every time you run tests. This won't cause any issues today, but its something I don't want to be copied for other tests. Can we use a dummy float value here?

Reply:
This comment goes for the other places `time.time()` is used in tests as well